cms/envs/common.py

```diff
@@ -766,6 +766,10 @@
     'notes',
 ]
 
+# XBlock types listed here will _always_ be selectable as Studio components
+XBLOCKS_ALWAYS_IN_STUDIO = [
```

Comment:
Maybe also just mention that this list will not appear in the Advanced Module list by default.